templates/examples/extensions/py_lambda_hooks/CanvasHook/CanvasHook.py

```diff
@@ -398,8 +411,13 @@ def handler(event, context):
                 result = query_grades_for_student(canvas, student_user_name, userinput)
                 return_message = result['Grades']
                 event['res']['session']['appContext']['altMessages']['markdown'] = return_message
+            else: 
+                return_message = 'There was an error processing your request. For a list of available options, type or say <i>canvas menu</i>.' 
+                event['res']['session']['appContext']['altMessages']['markdown'] = return_message
         except ValueError as e:
-            return_message = str(e)
-  
+            print ("ERROR: "+ str(e))  #print the exception
```

Comment:
This module is using print instead of logging. Let's take a backlog task for later to change prints to log messages.